traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/ParkingItem.java

```diff
@@ -45,6 +45,10 @@
         this.providerTranslationAttribution = providerTranslationAttribution;
     }
 
+    public BookingItem toBookingItem() {
+        return new BookingItem(id, title, subTitle, imageURL, price, productType, categories,
+                coordinate, providerTranslationAttribution, true, false);
```

Comment:
Whats going on here? Why are you changing a ParkingItem to a BookingItem?

This doesnt sound right

Reply:
this has to do with using the parking action strip, definitely worth streamlining. I will merge Parking, Bookable and Buyable ActionStripFragments into one and reconsider the params.

Reply:
✓